react/modules/tasks/nextjs-ssr-ssg.md

```diff
@@ -4,7 +4,7 @@
 
 1. Create a separate branch for this task from the previous branch task. Branch name: "nextjs-ssr-pages-api".
 2. Next.js Pages Api Integration:
-   - Migrate your application from vite to next.js by using Pages folder API.
+   - [Migrate your application from vite to next.js by using Pages folder API](https://nextjs.org/docs/pages/building-your-application/upgrading/from-vite).
```

Comment:
This is wrong link
![image](https://github.com/user-attachments/assets/9b1a584a-d9f9-466d-a82e-d059ac2a433f)

better use https://www.geeksforgeeks.org/next-js-upgrading-migrating-from-vite/?ysclid=m7dms13nb6414592988#steps-to-migrate-from-vite